config/logicsoft/ZDB5100.xml

```diff
@@ -0,0 +1,503 @@
+      <Help>
+			Pushbutton input debounce time in 0.01 seconds resolution.
+			  1-255:  1–2.55 seconds. Default is 5, which equals to a debounce-filter on 50 milliseconds (0.05 seconds).
+			</Help>
+    </Value>
+    <Value genre="config" index="12" instance="1" label="Pushbutton press threshold time" max="255" min="1" size="1" type="byte" value="20">
+      <Help>
+			Specifies the time that apushbuttonmust be activated before it is detected as pressed. Resolution is in 0.01 seconds.
+			  1-255:  1–2.55 seconds. Default is 20, which equals to 200 milliseconds (0.2 seconds).
+			</Help>
+    </Value>
+    <Value genre="config" index="13" instance="1" label="Pushbutton held threshold time" max="255" min="1" size="1" type="byte" value="50">
+      <Help>
+			Specifies the time that a pushbutton must have been activated before it is accepted as “held-down”. Resolution is 0.01 seconds.
+			  1-255:  1–2.55 seconds. Default is 50, which equals to 500 milliseconds (0,5 seconds).
+			</Help>
+    </Value>
+    <Value genre="config" index="14" instance="1" label="Global brightness control" size="4" type="int" value="4294967040">
+      <Help>        
+        Byte 1: Red brightness
+            0-255 Brightness level for the red colour in the 4 indicator groups (Default is 255).
+        Byte 2: Green brightness
+            0–255 Brightness level for the green colour in the 4 indicator groups (Default is 255).
+        Byte 3: Blue brightness
+            0-255 Brightness level for the blue colour in the 4 indicator groups (Default is 255).
+        Byte 4: Not used–must be set to 0.
+        Default: 4294967040
+			</Help>
+    </Value>
+    <Value genre="config" index="15" label="Associations groups, transmission when included secure" size="2" type="bitset" bitmask="8191" value="8191">
+      <Help>This parameter specifies if commands are transmitted as a secure message for each of the association groups.
+        Bitmask:
+        0: All messages in all groups are sentas insecure.
+        1: Messages in association group 2 aresentas secure.
+        2: Messages in association group 3 aresentas secure.
+        4: Messages in association group 4 aresentas secure.
+        8: Messages in association group 5 aresentas secure.
+        16: Messages in association group 6 aresentas secure.
+        32: Messages in association group 7 aresentas secure.
+        64: Messages in association group 8 aresentas secure.
+        128: Messages in association group 9 aresentas secure.
+        256: Messages in association group 10 aresentas secure.
+        512: Messages in association group 11 aresent as secure.
+        1024: Messages in association group 12 aresent as secure.
+        2048: Messages in association group 13 aresent as secure.
+        4096: Messages in association group 14 aresent as secure.
+        8191: Messages in allassociation groups are sent as secure. (Default)
+      </Help>
+      <BitSet id="1">
+        <Label>Bit 0</Label>
```

Comment:
Best if you can use a better labels for each Bits

Reply:
Right on. Labels are updated.